web/app/scripts/state/mapstate-service.js

```diff
@@ -40,7 +40,7 @@
          * @param {number} mapZoom The zoom level of the leaflet map
          */
         function setZoom(mapZoom) {
-            $log.debug('Setting zoom to: ' + JSON.stringify(mapZoom));
+            //$log.debug('Setting zoom to: ' + JSON.stringify(mapZoom));
```

Comment:
Forgot I did this lol.  Disabled them because it was getting in the way of my debug messages.  I'll get rid of the rest.